W-11792381: Improve the documentation about the upgrade process (#2357)
mulesoft/docs-mule-runtime-3: modules/ROOT/pages/mule-upgrade-tool.adoc

To upgrade or update your Mule instance to the latest version:

. Navigate to the `$MULE_HOME/tools` directory, where you downloaded the Mule upgrade tool.
. Run the `upgrade -n` subcommand, specifying as an argument the absolute path of the directory where the latest Mule runtime distribution is downloaded and unpacked, for example:
+

[source,bash,linenums]
----
$MULE_HOME/tools/upgrade-tool upgrade -n /<newMuleDistributionAbsolutePath>
----

[IMPORTANT]

> The upgraded Mule runtime will be in place of the previous version, this means its location will be `$MULE_HOME`.

Reviewer: ```suggestion
The upgraded Mule runtime instance replaces the previous version because its target location is `$MULE_HOME`.
```
